packages/dso-toolkit/src/components/list-button/list-button.mixins.scss

```diff
@@ -147,6 +148,10 @@
     }
   }
 
+  label {
+    cursor: inherit;
```

Comment:
Waar is deze voor?

Reply:
Ah! Leuk eigenlijk om daar `inherit` voor te gebruiken ipv. dubbele waarden

Reply:
Ja, zo hoef je er maar 1 te wijzigen